src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final User EXISTING_USER = new User("BobAlison", "password", 25);
+    private static final User NEW_USER_WITH_EXISTING_LOGIN = new User(
+            "BobAlison", "password", 30);
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH = 6;
+    private static final User NULL_USER = null;
+    private static final User VALID_USER = new User("NickAlison", "password", 65);
```

Comment:
What happens if I change minimum age in your program from 18 to 20? Will the test fail?